install_config/install/advanced_install.adoc

```diff
@@ -1181,6 +1181,139 @@ xref:../../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#i
 openshift_hosted_logging_storage_kind=dynamic
 ----
 
+[[enabling-service-catalog]]
+=== Enabling the Service Catalog
+
+[NOTE]
+====
+The service catalog is currently a Technology Preview feature.
+====
+
+Enabling the
+xref:../../architecture/service_catalog/index.adoc#architecture-additional-concepts-service-catalog[service catalog] allows service brokers to be registered with the catalog. The web
+console is also configured to enable an updated landing page for browsing the
+catalog.
+
+To enable the service catalog, add the following in your inventory file's
+`[OSEv3:vars]` section:
+
+----
+openshift_enable_service_catalog=true
+ifdef::openshift-enterprise[]
+openshift_service_catalog_image_prefix=openshift3/ose-
+endif::[]
+ifdef::openshift-origin[]
+openshift_service_catalog_image_prefix=openshift/origin-
+openshift_service_catalog_image_version=latest
+endif::[]
+----
+
+When the service catalog is enabled, the web console shows the updated landing
+page but still uses the normal image stream and template behavior. To deploy and
+register the provided service brokers, see
+xref:configuring-ansible-service-broker[Configuring the Ansible Service Broker]
+and xref:configuring-template-service-broker[Configuring the Template Service Broker].
+
+[[configuring-ansible-service-broker]]
+=== Configuring the Ansible Service Broker
+
+[NOTE]
+====
+The Ansible service broker is currently a Technology Preview feature.
+====
+
+If you have xref:enabling-service-catalog[enabled the service catalog], you can
+also enable the
+xref:../../architecture/service_catalog/ansible_service_broker.adoc#arch-ansible-service-broker[Ansible service broker] (ASB).
+
+The ASB deploys its own etcd instance separate from the etcd used by the rest of
+the {product-title} cluster. The ASB's etcd instance requires separate storage
+using persistent volumes (PVs) to function, as well as for the Ansible playbook
+bundles (APBs) that are created using the ASB. If no PV is available, etcd will
```

Comment:
Adding to what @fabianvf wrote.  

The ASB requires it's own etcd, etcd needs a PV so it's data can be persisted.  We store state information in etcd for the broker.  That is one PV that is required.  With this PV, etcd will not come up.  If our etcd does not come up, the ASB pod will not come up.  

In addition, as @fabianvf mentioned some APBs may also require a PV for their own usage.
We have 2 APBs in 3.6.0.  Mediawiki and PostgreSQL.  Both of these require their own PV to deploy.

If a user deploys Mediawiki, the mediawiki deployment will create a PVC that needs to be filled for the application to come up.  Same for PostgreSQL.

To be clear, we don't actually store APBs themselves on a PV, it's not like we are downloading and caching them.  An APB is a regular image stored in a registry and run as a container.  It's not treated differently.